src/adaXT/predict/predict.pyi

```diff
@@ -36,7 +38,14 @@ class Predict:
         pass
 
     @staticmethod
-    def forest_predict(predictions: np.ndarray, **kwargs):
+    def forest_predict(
+        X_old: np.ndarray,
+        Y_old: np.ndarray,
+        X_new: np.ndarray,
```

Comment:
change these to `cnp.ndarray`?

Reply:
the pyi file is simply an outwards facing file for the language server. Has no input, but is not cython defined, so can not make use of cnp.